config/core/oracle.json

```diff
@@ -417,6 +417,22 @@
         "maxSubscriptionsPerConnection": 0
       },
       "type": "price_provider"
+    },
+    {
+      "name": "marketmap_api",
+      "api": {
+        "enabled": true,
+        "timeout": 20000000000,
+        "interval": 10000000000,
+        "reconnectTimeout": 2000000000,
+        "maxQueries": 1,
+        "atomic": true,
+        "url": "blockchain:9090",
```

Comment:
This works in our docker compose, but I think it's a weird default to have otherwise. It won't work if people are running the sidecar via systemd.